app/Models/User/Study/Bookmark.php

```diff
@@ -141,6 +144,11 @@ public function book()
         );
     }
 
+    public function bibleBook()
+    {
+        return $this->hasOne(BibleBook::class, ['book_id', 'bible_id'], ['book_id', 'bible_id']);
```

Comment:
@bradflood Do you mean if both entities will be in the same SQL statement? Because the answer is no. Laravel will execute two different queries when it tries to fetch the Bookmarks and the BibleBook. The first query will hit the `dbp_users` database, and the second will hit the `dbp` database, as you can see in the following screenshot:

![Screenshot from 2023-04-04 14-33-32](https://user-images.githubusercontent.com/73488660/229904621-99eb11dd-adea-4874-beba-119a40f32384.png)